modules/support-collecting-host-network-trace.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="support-collecting-host-network-trace_{context}"]
+= Collecting a host network trace
+
+Sometimes, troubleshooting a network-related issue is simplified by tracing network communication and capturing packets on multiple nodes at the same time.
+
+You can use a combination of the `oc adm must-gather` command and the `registry.redhat.io/openshift4/network-tools-rhel8` container image to gather packet captures from nodes.
+You can analyze the packet captures to understand network communication and assist with diagnosing network issues.
+
+The `oc adm must-gather` command is used to run the `tcpdump` command in pods on specific nodes.
+The `tcpdump` command records the packet captures in the pods.
+When the `tcpdump` command exits, the `oc adm must-gather` command transfers the files with the packet captures from the pods to your client machine.
+
+[TIP]
+====
+The sample command in the following procedure demonstrates performing a packet capture with the `tcpdump` command.
+However, you can run any command in the container image that is specified in the `--image` argument to gather troubleshooting information from multiple nodes at the same time.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Start a packet capture from the host network on some nodes:
++
+[source,terminal]
+----
+$ oc adm must-gather \
+    --dest-dir /tmp/captures \  <.>
```

Comment:
I don't recall what buggers it.  It's Rougify that does that work.  Syntax highlighting is a secondary consideration because access.redhat.com does _no_ syntax highlighting.